plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -40,20 +40,9 @@ GSDevice11::GSDevice11()
 	m_state.bf = -1;
 
 	m_mipmap = theApp.GetConfigI("mipmap");
-
-	if (theApp.GetConfigB("UserHacks"))
-	{
-		UserHacks_unscale_pt_ln = theApp.GetConfigB("UserHacks_unscale_point_line");
-	}
-	else
-	{
-		UserHacks_unscale_pt_ln = false;
-	}
 }
 
-GSDevice11::~GSDevice11()
-{
-}
+GSDevice11::~GSDevice11() { }
```

Comment:
Don't see why the destructor has been reformatted.

Reply:
Thought it was better since there's no code inside it but sure I'll change both of them back.